lib/potluck.rb

```diff
@@ -0,0 +1,67 @@
+require "pry"
+
+class Potluck
+
+  attr_reader :date, :dishes
+  attr_accessor :menu
+
+  def initialize(date)
+    @date = date
+    @dishes = []
+    @menu_hash = {}
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
+  def get_all_from_category(food_type)
+    all = @dishes.find_all do |food|
```

Comment:
You could refactor this by removing the variable `all`. You are setting the return value of the `find_all` to a variable and then returning the variable. Just return the return value of `find_all`